index.js

```diff
@@ -233,8 +233,8 @@ exports.deleteAccount = async function (options) {
             throw e;
         }
     }
-    
-    if (await confirmDelete()) {
+
+    if (options.force || await confirmDelete()) {
```

Comment:
It would be cool to add a note about this flag in README.
Small test is also appritiated.

Reply:
Added note on README. I tried to run integration tests locally, but doing `yarn test` is insufficient. I guess I need to spin off a NEAR node locally to make it work.

Reply:
It should work with testnet, but ok.